CleverTapSDK/InApps/CustomTemplates/SystemAppFunction/CTPushPermissionSystemAppFunction.m

```diff
@@ -0,0 +1,28 @@
+//
+//  CTPushPermissionSystemAppFunction.m
+//  CleverTapSDK
+//
+//  Created by Nishant Kumar on 20/03/25.
+//  Copyright © 2025 CleverTap. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "CTPushPermissionSystemAppFunction.h"
+#import "CTAppFunctionBuilder-Internal.h"
+#import "CTSystemAppFunctionPresenter.h"
+#import "CTConstants.h"
+
+@implementation CTPushPermissionSystemAppFunction
+
++ (CTCustomTemplate *)buildTemplateWithHanlder:(CTSystemTemplateActionHandler *)systemTemplateActionHandler {
+    CTCustomTemplateBuilder *pushPermissionbuilder = [[CTAppFunctionBuilder alloc] initWithIsVisual:NO isSystemDefined:YES];
+    [pushPermissionbuilder setName:CLTAP_PUSH_PERMISSION_TEMPLATE_NAME];
+    [pushPermissionbuilder addArgument:CLTAP_FB_SETTINGS_KEY withBool:YES];
```

Comment:
The fallback settings default value is `NO` on Android.

Reply:
Done.